cypress/integration/ProcessorTest.ts

```diff
@@ -0,0 +1,342 @@
+        .should("be.visible")
+        .select(source[0]);
+      cy.ouiaId("missing-source-parameters", "PF4/TextInput").should(
+        "not.exist"
+      );
+      cy.ouiaId("channel", "PF4/TextInput")
+        .should("be.visible")
+        .type(source[1]);
+      cy.ouiaId("token", "PF4/TextInput").should("be.visible").type(source[2]);
+
+      //Filters
+      cy.ouiaId("add-filter", "PF4/Button").should("be.visible").click();
+      cy.ouiaId("item-0").within(() => {
+        cy.ouiaId("filter-key", "PF4/TextInput").type(filters[0][0]);
+        cy.ouiaId("filter-type", "PF4/FormSelect").select("String equals");
+      });
+      //The filter-value was detached from DOM and we need to find the context again.
+      cy.ouiaId("item-0")
+        .ouiaId("filter-value", "PF4/TextInput")
+        .type(filters[0][2]);
+      cy.ouiaId("item-1").within(() => {
+        cy.ouiaId("filter-key", "PF4/TextInput").type(filters[1][0]);
+        cy.ouiaId("filter-type", "PF4/FormSelect").select("String equals");
+      });
+      //The filter-value was detached from DOM and we need to find the context again.
+      cy.ouiaId("item-1")
+        .ouiaId("filter-value", "PF4/TextInput")
+        .type(filters[1][2]);
+
+      //Transformation is related to action and it is not a part of Source processor
+
+      cy.ouiaId("submit").click();
+
+      cy.ouiaId("loading-table", "PF4/Card", { timeout: 30000 }).should(
+        "not.exist"
+      );
+
+      /*
+       * It is necessary to check data in the same test.
+       * The mocked data are deleted when the page is refreshed.
+       */
+
+      cy.ouiaId("Processors list table", "PF4/Table")
+        .ouiaId(processorName, "PF4/TableRow")
+        .should("be.visible")
+        .within(() => {
+          cy.get("td")
+            .eq(3)
+            .then(($state) => {
+              cy.wrap($state).should("have.text", "accepted");
+              cy.wrap($state, { timeout: 15000 }).should(
+                "have.text",
+                "provisioning"
+              );
+              cy.wrap($state, { timeout: 30000 }).should("have.text", "ready");
+            });
+          cy.get("td").eq(0).should("have.text", processorName).click();
+        });
+
+      cy.ouiaId("processor-name", "PF4/Text").should(
+        "have.text",
+        processorName
+      );
+
+      cy.get("[class='pf-c-description-list__description']")
+        .should("have.length", source.length)
+        .each((element, index) => {
+          cy.wrap(element).should("have.text", source[index]);
+        });
+
+      cy.ouiaId("filters", "PF4/Table").within(() => {
+        filters.forEach((item, rowIndex) => {
+          cy.ouiaId(item[0], "PF4/TableRow")
+            .find("td")
+            .each((cell, cellIndex) => {
+              cy.wrap(cell).should("have.text", filters[rowIndex][cellIndex]);
+            });
+        });
+      });
+    });
+  });
+
+  describe("Edit Sink Processors", () => {
```

Comment:
I think the only supported edit operation, for processors, is a change of processor filters. Please double check it with team. If answer is yes. We should remove some tests, for example edit processor name.

Reply:
After https://issues.redhat.com/browse/MGDOBR-731 sure, we need to cover with some IT tests about specific filter types.
AFAIK, the processor name change is supported

Reply:
Processor name editing was working fine when I reviewed #54. I've tested it against the dev APIs. So something has changed. Let's check this next week.

Reply:
I checked and I was wrong. Processor names cannot be changed right now.